org.speed.dreams.yaml

```diff
@@ -0,0 +1,166 @@
+app-id: org.speed.dreams
+runtime: org.freedesktop.Platform
+runtime-version: '1.6'
+sdk: org.freedesktop.Sdk
+command: speed-dreams-2
+separate-locales: false
+finish-args: 
+  - --socket=wayland
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --filesystem=home
+  - --socket=pulseaudio
+  - --share=network
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.a'
+  - '*.la'
+  
+modules:
+  - shared-modules/glu/glu-9.0.0.json
+
+  - name: giflib
+    config-opts:
+      - --prefix=/app
+      - --disable-static
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/sourceforge/giflib/giflib-5.1.4.tar.bz2
+        md5: 2c171ced93c0e83bb09e6ccad8e3ba2b
+  
+  - name: freeglut
+    buildsystem: cmake
```

Comment:
`cmake-ninja` is preferred wherever possible, because of its superior build times.